specs/cm/change-mgt-vocab.ttl

```diff
@@ -28,10 +28,15 @@
 oslc_cm:
   a owl:Ontology ;
   dcterms:title "The OSLC Change Management(CM) Vocabulary" ;
+  rdfs:label "Change Management(CM)" ;
```

Comment:
As Nick suggested in my PR, let's have the same `title` and `label`.

Reply:
Fixed, made all dcterms:title and rdfs:label content the same.